apps/services/user-profile/README.md

```diff
@@ -3,6 +3,7 @@
 ## About
 
 A service that is responsible for providing and storing User profile info. With the users National Registration id stores contact info and preferred locale. Additional responsibility is a verification process of email and mobile phone.
+The user-profile table is not prepopulated with everyone in Iceland, therefore due to multiple onboarding-flows the user-profile can be created under a variety of circumstances, you will see the findOrCreateUserProfile method a variety of places in this project's code.
```

Comment:
I think it is not necessary to  write in the documentation which functions are used internally.
```suggestion
The user-profile table is not prepopulated with everyone in Iceland, therefore due to multiple onboarding-flows the user-profile can be created under a variety of circumstances. If a user profile is not found in the user-profile table it is created from data from <insert data origin>, and thenceforth used.
```